Sources/BFKit/Linux/Foundation/ArrayExtension.swift

```diff
@@ -35,6 +35,45 @@ import Foundation
 public extension Array {
     // MARK: - Functions
     
+    /// Simulates the array as a circle. When it is out of range, begins again.
+    ///
+    /// - Parameter index: The index.
+    /// - Returns: Returns the object at a given index.
+    public func circleObject(at index: Int) -> Element {
+        return self[self.superCircle(at: index, size: self.count)]
+    }
+    
+    /// Randomly selects an element from self and returns it.
+    ///
+    /// - returns: An element that was randomly selected from the array.
+    public func random() -> Element {
+        let arrayCount = UInt32(self.count)
+        #if os(Linux)
```

Comment:
Here we could use [randomInt](https://github.com/FabrizioBrancati/BFKit-Swift/blob/master/Sources/BFKit/Linux/Foundation/NumberExtension.swift#L56) from number extension